less/components/popups.less

```diff
@@ -69,12 +67,54 @@
     align-items: center;
     width: 100%;
     height: 80px;
-    min-height: 80px;
+    flex-shrink: 0;
```

Comment:
can we use flex for this? I thought we had to change away from flex to get IE working properly